decryption_utils/Python/decrypt.py

```diff
@@ -82,7 +82,8 @@ def get_private_key(loc):
         return RSA.importKey(fh.read())
 
 def valid_uuid(filename):
-    uuid = filename.split("/")[-1].split(".")[0]
+    head, tail = os.path.split(filename)
```

Comment:
Huge nitpick, but reading this I was not sure what `head` and `tail` were supposed to represent.
Recommend renaming to something like `path` and `file` or similar.